testsuite/MDAnalysisTests/test_persistence.py

```diff
@@ -36,10 +36,8 @@
 import shutil
 import warnings
 
-"""These tests need to be reworked for the new 363 topoloy system
 
-"""
-class SkipAtomGroupPickle(object):
+class TestAtomGroupPickle(object):
```

Comment:
The hash way of seeing things is neater than what was in place before (even if functionally equivalent). In that sense `is_anchor` could be made to affect even named universes
(one then may ask the question of why giving a name if not going to use as anchor, but let's leave that up to the user.)